infra/app/env-template/variables.tf

```diff
@@ -45,12 +45,23 @@ variable "participant_s3_presigned_url_expiration" {
   default     = "604800"
 }
 
+variable "participant_s3_presigned_url_renewal_threshold" {
+  type        = string
+  description = "The number of seconds a presigned s3 url is active before it is renewed"
+  default     = 7 * 60 * 60 # 25200 seconds = 7 hours
```

Comment:
.. we are refreshing these every 3 hours with a threshold of 7? 
Do we know why a week is not working? :concerned:

Reply:
Yah. It's in the PR context